api/v1alpha1/mariadb_types.go

```diff
@@ -345,9 +349,15 @@ func (m *MariaDB) SetDefaults(env *environment.Environment) {
 	if m.Spec.Image == "" {
 		m.Spec.Image = env.RelatedMariadbImage
 	}
-	if m.Spec.RootPasswordSecretKeyRef == (corev1.SecretKeySelector{}) {
+
+	if m.Spec.RootEmptyPassword == nil {
+		defaultRootEmptyPassword := false
+		m.Spec.RootEmptyPassword = &defaultRootEmptyPassword
```

Comment:
The [k8s.io/utils/ptr](https://pkg.go.dev/k8s.io/utils/ptr) pkg is a very nice utility I am trying to migrate to progresively, can we use it here?
```go
ptr.To(false)
```
See:
https://github.com/mariadb-operator/mariadb-operator/blob/d2ee5de461323cd5eb29b22b8952a39b0cc5ebb0/api/v1alpha1/mariadb_types_test.go#L82

Reply:
Done